package.json

```diff
@@ -16,6 +16,7 @@
     "eslintconfig"
```

Comment:
Wondering if it's a good time to convert this repo to `"type": "module"` to avoid having to use `.mjs` extensions on the new files? It would mean converting the others to use proper exports instead of `module.exports`.

If we can, then we could add the new config exports to `index.js` (which is the main entrypoint) so that consumers could go `import { litConfig } from 'eslint-config-brightspace';`.

Reply:
I was testing this out and eslint8 seems to use commonjs to import the shared config, so `type: module` would be a breaking change.

Reply:
Gotcha. Maybe a breaking change and major version bump is OK then? Then we could actually just ditch those old files (moving them into the new ones).

Reply:
I'd be ok with that. 😄